resources/com/mesosphere/global/terraform_file_deploy.sh

```diff
@@ -100,20 +100,40 @@ EOF
   fi
   cat <<EOF > marathon-lb-options.json
 {
-    "marathon-lb": {
-        "secret_name": "marathon-lb/service-account-secret",
-        "marathon-uri": "https://marathon.mesos:8443"
-    }
+  "marathon-lb": {
+    "auto-assign-service-ports": true,
+    "bind-http-https": true,
+    "cpus": 2,
+    "haproxy_global_default_options": "redispatch,http-server-close,dontlognull",
+    "haproxy-group": "external",
+    "haproxy-map": true,
+    "instances": 1,
+    "mem": 1024,
+    "minimumHealthCapacity": 0.5,
+    "maximumOverCapacity": 0.2,
+    "name": "marathon-lb",
+    "parameters": [],
+    "role": "slave_public",
+    "strict-mode": false,
+    "sysctl-params": "net.ipv4.tcp_tw_reuse=1 net.ipv4.tcp_fin_timeout=30 net.ipv4.tcp_max_syn_backlog=10240 net.ipv4.tcp_max_tw_buckets=400000 net.ipv4.tcp_max_orphans=60000 net.core.somaxconn=10000",
+    "container-syslogd": false,
+    "max-reload-retries": 10,
+    "reload-interval": 10,
+    "template-url": "",
+    "marathon-uri": "http://marathon.mesos:8080",
+    "secret_name": ""
+  }
 }
 EOF
-  "${TMP_DCOS_TERRAFORM}"/dcos package install --yes --options=marathon-lb-options.json marathon-lb > /dev/null 2>&1 || exit 1
-  timeout -t 120 bash <<EOF || ( echo -e "\e[31m failed to deploy marathon-lb... \e[0m" && exit 1 )
-while ${TMP_DCOS_TERRAFORM}/dcos marathon task list --json | jq .[].healthCheckResults[].alive | grep -q -v true; do
```

Comment:
please do not change

Reply:
fixed at 1bd4b9127b8d219de1e3ce02dd03ee08e36041f3